Classes/Driver/AmazonS3Driver.php

```diff
@@ -622,7 +642,7 @@ public function deleteFile($fileIdentifier)
      * @return string
      * @throws InvalidPathException
      */
-    public function hash($fileIdentifier, $hashAlgorithm)
+    public function hash($fileIdentifier, $hashAlgorithm): string
```

Comment:
`sha1_file` en `md5_file` kunnen false teruggeven

Reply:
Dit is verholpen door een exception op te gooien als `sha1_file` of `md5_file` false als waarde heeft.